memory_ttl_test.go

```diff
@@ -29,6 +29,32 @@ func TestMemoryCacheTTL(t *testing.T) {
 	}
 }
 
+func TestMemoryCacheTTLGetExpired(t *testing.T) {
+	// Needs go test -race to catch problems
```

Comment:
We already have race in testing script https://github.com/koding/cache/blob/master/.travis.yml#L15 Did you mean something else @philpearl ?